### ISSUE TITLE
Qt: "QNetworkReplyHttpImplPrivate::_q_startOperation was called more than once" prior to Qt 5.12.2

### ISSUE THREAD

**Issue by Echelon9** [radareorg/cutter#2051]
**Environment information**
* Operating System: Linux
* Cutter version: 1.10.0
* Obtained from:
  - [ ] Built from source
  - [x] Downloaded from release
  - [ ] Distribution repository
* File format: Not applicable - bug is in a core Cutter UI action

Upstream dependency Qt fixed a bug in `QNetworkReplyHttpImplPrivate::_q_startOperation()` that can be triggered in Cutter.

In the Cutter console log, the following debug message is reported:
```
QNetworkReplyHttpImplPrivate::_q_startOperation was called more than once QUrl("https://api.github.com/repos/radareorg/cutter/releases/latest")
```

Updating Cutter's Qt dependency to >= 5.12.2 will resolve this.

**To Reproduce**

1. Go to 'Help' top-menu item, then click on 'About' sub-menu item
2. Click on 'Check for updates' button in the modal window
4. See error in console log

**Expected behavior**

No debug message issued by Qt dependency.

**Screenshots**

n/a

**Additional context**

See upstream Qt bug report: https://bugreports.qt.io/browse/QTBUG-72463

**Comment by karliss** [comment]
Cutter release packages are currently compiled using Qt 5.14 so I assume this problem has been resolved. Reopen this issue if it isn't.

**Comment by Echelon9** [comment]
Confirmed fixed with [cutter v1.11.0](https://github.com/radareorg/cutter/releases/tag/v1.11.0)
